code/modules/mob/living/carbon/xenomorph/castes/bull/castedatum_bull.dm

```diff
@@ -119,3 +120,37 @@
 
 	// *** Defense *** //
 	soft_armor = list("melee" = 40, "bullet" = 50, "laser" = 40, "energy" = 40, "bomb" = XENO_BOMB_RESIST_0, "bio" = 33, "rad" = 33, "fire" = 50, "acid" = 33)
+
+/datum/xeno_caste/bull/primordial
+	upgrade_name = "Primordial"
+	caste_desc = "A fast and agile hit-and-runner that nothing will stop." //Help
+	primordial_message = ""
```

Comment:
Bull
Desc: "Bloodthirsty horned devil of the hive. Stay away from its path."
Message: "We are the spearhead of the hive. Run them all down."
```suggestion
/datum/xeno_caste/bull/primordial
	upgrade_name = "Primordial"
	caste_desc = "Bloodthirsty horned devil of the hive. Stay away from its path."
	primordial_message = "We are the spearhead of the hive. Run them all down."
```